tox.ini

```diff
@@ -98,6 +99,24 @@ commands =
     coverage html
     coverage xml
 
+[testenv:black-check]
+basepython = python3.7
+skip_install = True
+deps =
+    black
+
+commands =
+    black --line-length=79 --check --diff -v --skip-string-normalization bobtemplates setup.py
+
+[testenv:black-enforce]
+basepython = python3.7
+skip_install = True
+deps =
+    black
+
+commands =
+    black -v --line-length=79 --skip-string-normalization bobtemplates setup.py
```

Comment:
Just in case... When discussing the black PLIP (https://github.com/plone/Products.CMFPlone/issues/2754) it was decided to use black with all the defaults.

Reply:
I wanted that for the first run, as it confirms with flake8 settings. Will do more basic setting later to see different results.

Reply:
I don't want to have to many changes on one commit, I do it step by step to have black defaults. Before we did have already used black for some checks, but with those specific config.